moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h

```diff
@@ -119,11 +115,11 @@ class PlanningContext
 
   /** \brief Solve the motion planning problem and store the result in \e res. This function should not clear data
    * structures before computing. The constructor and clear() do that. */
-  virtual bool solve(MotionPlanResponse& res) = 0;
+  virtual void solve(MotionPlanResponse& res) = 0;
 
   /** \brief Solve the motion planning problem and store the detailed result in \e res. This function should not clear
    * data structures before computing. The constructor and clear() do that. */
-  virtual bool solve(MotionPlanDetailedResponse& res) = 0;
+  virtual void solve(MotionPlanDetailedResponse& res) = 0;
```

Comment:
@henningkayser I am wondering if we should get rid of this. OMPL is the only planner that uses this interface (and has distinguishable processing steps as far as I know) and in the planning pipeline we don't use it anyways. Rather than enforcing this interface on all planners we should consider a custom solution for OMPL